config_example.py

```diff
@@ -0,0 +1,18 @@
+"""music.py"""
+# Chats where the users can download music from video services
+# and additional users that need to use the music downloader
+# feature but are not in any chat
+MUSIC_CHATS = ['12345', '67890']
+MUSIC_USERS = [53257673]
+# leave it as an empty array
```

Comment:
Should be list of `str` ("username") or `int` (ID)

username is string and can't only be numbers. but it does not really matter since this is just for example configuration.